packages/4337-local-bundler/hardhat.config.ts

```diff
@@ -0,0 +1,8 @@
+import "@nomicfoundation/hardhat-toolbox"
```

Comment:
I don't know tsc worked without this

Reply:
Maybe it doesn't use some of the extensions that were causing problems in the other packages?